src/payment/coupon/data/service.js

```diff
@@ -1,23 +1,15 @@
 import pick from 'lodash.pick';
 
-import { handleRequestError } from '../../../common/serviceUtils';
+import { handleRequestError, applyConfiguration } from '../../../common/serviceUtils';
```

Comment:
"validateConfiguration" has been DRY'd up outside of this.  Using the common one saves me a test. ;)